ui/raidboss/data/05-shb/eureka/delubrum_reginae.js

```diff
@@ -1082,14 +1081,11 @@ export default {
         data.unseenBadRows.sort();
         data.unseenBadCols.sort();
       },
-      alertText: (data, _, output) => {
+      alertText: (data, _matches, output) => {
         delete data.unseenIds;
 
-        const rows = data.unseenBadRows;
-        const cols = data.unseenBadCols;
-
         // consider asserting that badCols are 0, 2, 4 here.
-        if (rows.includes(2))
+        if (data.unseenBadRows.includes(2))
```

Comment:
Seems reasonable.  As an explanation, this code was here mostly to match the savage version as a direct copy/paste.  I think this is clearer though and is fine.